### PR TITLE
[Tests-Only]  Added default capabilities scenario for a normal user

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
Since admin user cannot get capabilities detail from ocis-reva server but normal user can, test scenario is added for a normal user to check default capabilities set.

## Related Issue
- https://github.com/owncloud/ocis-reva/issues/149

## How Has This Been Tested?

- :robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by kiranparajuli589** [comment]
different behaviour of `ocis-reva` and `core` servers for capabilities response is described in issue https://github.com/owncloud/ocis-reva/issues/175

**Comment by individual-it** [comment]
Please make a separate test for OCIS and one for oC10, so that both path in the respected environment

File: tests/acceptance/features/apiCapabilities/capabilitiesWithNormalUser.feature [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  Scenario: getting default capabilities with normal user
+    When user "user0" retrieves the capabilities using the capabilities API
+    Then the capabilities should contain
+      | capability    | path_to_element                           | value             |
+      | core          | pollinterval                              | 60                |
+      | core          | webdav-root                               | remote.php/webdav |
+      | core          | status@@@edition                          | %edition%         |
+      | core          | status@@@productname                      | ownCloud          |
+      | core          | status@@@version                          | 10.5.0.0          |
+      | core          | status@@@versionstring                    | 10.5.0alpha1      |
```
Comment (individual-it) [inline tests/acceptance/features/apiCapabilities/capabilitiesWithNormalUser.feature:61]: for owncloud we should be able to get these values dynamically

File: tests/acceptance/features/apiCapabilities/capabilitiesWithNormalUser.feature [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      | files_sharing | user_enumeration@@@enabled                | 1                 |
+      | files_sharing | user_enumeration@@@group_members_only     | 1                 |
+      | files_sharing | user@@@send_mail                          | 1                 |
+      | files         | bigfilechunking                           | 0                 |
+      | files         | privateLinks                              | 0                 |
+      | files         | privateLinksDetailsParam                  | EMPTY             |
+
+  # remove this scenario after fixing tagged issue as its just created to show difference
+  # in the response items in different environment (core & ocis-reva)
+  @skipOnOcis @issue-ocis-reva-176
```
Comment (individual-it) [inline tests/acceptance/features/apiCapabilities/capabilitiesWithNormalUser.feature:51]:
```suggestion
  @skipOnOcis @issue-ocis-reva-175 @issue-ocis-reva-176
```

File: tests/acceptance/features/apiCapabilities/capabilitiesWithNormalUser.feature [hunk 3]
```diff
@@ -0,0 +1,90 @@
+@api @TestAlsoOnExternalUserBackend @files_sharing-app-required
+Feature: default capabilities for normal user
+
+  Background:
+    Given using OCS API version "1"
+    And user "user0" has been created with default attributes and without skeleton files
+
+  @skipOnOcV10 @issue-ocis-reva-176
```
Comment (individual-it) [inline tests/acceptance/features/apiCapabilities/capabilitiesWithNormalUser.feature:8]:
```suggestion
  @skipOnOcV10 @issue-ocis-reva-175 @issue-ocis-reva-176
```

**Review by individual-it — CHANGES_REQUESTED** [review]

**Comment by individual-it** [comment]
please also make a PR to ocis-reva and ocis repo that uses this branch as test-runner to make sure the tests pass there

**Comment by kiranparajuli589** [comment]
https://github.com/owncloud/ocis/pull/255 - run tests on ocis

**Comment by kiranparajuli589** [comment]
https://github.com/owncloud/ocis-reva/pull/178 - run tests on ocis-reva repo

**Comment by kiranparajuli589** [comment]
@individual-it changes has been made, please review again!

**Review by individual-it — APPROVED** [review]

**Pull request merged**